src/engine/srv.c

```diff
@@ -440,9 +440,15 @@ dss_srv_handler(void *arg)
 			D_GOTO(nvme_fini, rc = dss_abterr2der(rc));
 		}
 
+#ifdef ULT_MMAP_STACK
```

Comment:
Might I suggest doing something like this in a header file (don't commit this here)
```suggestion
#ifdef ULT_MMAP_STACK
#define daos_abt_thread_create mmap_stack_thread_create
#else 
#define daos_abt_thread_create ABT_thread_create
#endif
```